data/model/repository.py

```diff
@@ -733,16 +733,16 @@ def force_cache_repo_size(repo_id: int):
     try:
         now_ms = get_epoch_timestamp_ms()
         subquery = (
-            Tag.select(Tag.repository_id)
+            Tag.select(Tag.manifest)
             .where(Tag.hidden == False)
             .where((Tag.lifetime_end_ms >> None) | (Tag.lifetime_end_ms > now_ms))
```

Comment:
Its missing on both sides but lets add a repository_id where clause here in the subquery as well since we are here and I think that will make it more efficient